devito/operator.py

```diff
@@ -155,6 +156,13 @@ def __init__(self, expressions, **kwargs):
         # autotuning reports, etc
         self._state = {}
 
+        # Form and gather any required implicit expressions.
+        # Implicit expressions are those not explicitly defined by the user
+        # but instead are requisites of some specified functionality. Since they
+        # must be treated in a similar manner to user specified expressions
+        # they are added prior to the creation of the IET.
```

Comment:
"prior to the creation of the IET". In fact the IET is created much, much later. So I'd say "prior to expression lowering"

Reply:
Thanks. Updated.